security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -12,6 +12,21 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-48"]
+== OpenShift Compliance Operator 0.1.48
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.48:
+
+* link:https://access.redhat.com/errata/RHBA-2022:xxxx[RHBA-2022:xxxx - OpenShift Compliance Operator bug fix and enhancement update]
+
+[id="openshift-compliance-operator-0-1-48-bug-fixes"]
+
+=== Bug fixes
+
+* Previously, some rules associated with extended Open Vulnerability and Assessment Language (OVAL) definitions would have a `checkType` of `None`. This is because the compliance Operator was not processing extended OVAL definitions when parsing rules. With this update, content from extended OVAL definitions is parsed so that these rules now have a `checkType` of either `Node` or `Platform`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2040282[*BZ#2040282*])
+
+* Previously, manually created `KubeletConfig` `MachineConfig` objects would prevent a machine config from being generated for a remediation, leaving the remediation in the `Pending` state.  With this release, a new `KubeletConfig` `MachineConfig` object is created by the `KubeletConfig` object generated by the remediation, regardless if there is a manually created `KubeletConfig` `MachineConfig` object. As a result, `KubeletConfig` remediations now work as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2040401[*BZ#2040401*])
```

Comment:
Going along with Lance's suggestion, I think it should be...

* Previously, manually created `KubeletConfig` objects prevented a machine config object from being generated for remediation, which left the remediation in the `Pending` state.  With this release, a new `KubeletConfig` object is created by the `KubeletConfig` object generated by the remediation, regardless if there is a manually created `KubeletConfig` object. As a result, `KubeletConfig` remediations now work as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2040401[*BZ#2040401*])

Reply:
Hi @stevsmit This should be:

Previously, manually created `Machine Config` objects for `KubeletConfig` prevented a machine config object from being generated for remediation

Reply:
Minor adjustment to the intro sentence to help with parallelism:
 Previously, **a** manually created `MachineConfig` **object** for `KubeletConfig` prevented a `KubeletConfig` object from being generated for remediation, leaving the remediation in the `Pending` state.